src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/dllmain.cpp

```diff
@@ -120,6 +121,10 @@ CreateApplication(
         g_fInProcessApplicationCreated = true;
 
         std::unique_ptr<IN_PROCESS_APPLICATION, IAPPLICATION_DELETER> inProcessApplication;
+
+        // TODO not sure how easy it will be to untangle errors here
+        // ErrorContext errorContext;
```

Comment:
Did you figure out if it's possible?

Reply:
It is possible, but it isn't an important scenario. There are a few errors that can occur here, but they are very rare because we already invoked hostfxr earlier. After main has started, we would return the developer exception page.